NineChronicles.RPC.Shared/Services/IBlockChainService.cs

```diff
@@ -16,5 +16,7 @@ public interface IBlockChainService : IService<IBlockChainService>
         UnaryResult<bool> SetAddressesToSubscribe(IEnumerable<byte[]> addressesBytes);
 
         UnaryResult<bool> IsTransactionStaged(byte[] txidBytes);
+
+        UnaryResult<bool> PutException(string code, string message);
```

Comment:
How about rename it to something like `ReportException`? And code parameter would be better if it became an integer or byte value.

Reply:
In the `nekoyume-unity`, the `code` type is a string. I intended to use the type without casting it. If using `byte[]` or `int` instead of a string, also need `message` parameter using `byte[]`?